.env

```diff
@@ -1,2 +1,4 @@
 PORT=3000
-NODE_ENV=12
\ No newline at end of file
+NODE_ENV=12
```

Comment:
?

Reply:
my mistake. I need to force using this app in node version 12, as on newer versions there is an issue with sql package.

Reply:
you can indicate node version in package.json: https://docs.npmjs.com/files/package.json#engines
but afaik it's just a suggestion, that might not be respected by some PaaS platforms.

NODE_ENV is typically either "development" or "production". This flag could be used to e.g. differentiate between our "npm start" and "npm run dev".